src/product/product.controller.spec.ts

```
@@ -108,6 +110,58 @@ describe('ProductController', () => {
+    });
+
+    it('should handle NotFoundException from service', () => {
+      // Arrange
+      const productId = '999';
+      const updateProductDto: UpdateProductDto = {
+        name: '수정된 상품',
+      };
+
+      jest
+        .spyOn(service, 'update')
+        .mockImplementation(() => {
```

ESLint

[error] Replace `⏎········.spyOn(service,·'update')⏎········` with `.spyOn(service,·'update')`

(prettier/prettier)